### ISSUE TITLE
Statamic v4 Support

### ISSUE THREAD

**Issue by heywhy** [aryehraber/statamic-splash#21]
I'm unable to upgrade to v4 because of the requirement in the composer.json file. A new release is needed.

**Comment by aryehraber** [comment]
Hi @heywhy, thanks for opening this issue! 

Unfortunately, it isn't as simple as bumping the version in `composer.json`, some actual UI fixes need to be made before an upgrade can be released. I'll try and get to it asap but can't promise a timeline atm. Will update here once it is ready!

**Comment by aryehraber** [comment]
Just released v2.0.0 of Splash which adds Statamic 4 support. Please open a new issue if you encounter any issues, thanks!
